apis/v1alpha3/backendtlspolicy_types.go

```diff
@@ -235,3 +260,43 @@ const (
 	// Support: Core
 	URISubjectAltNameType SubjectAltNameType = "URI"
 )
+
+const (
+	// This reason is used with the "Accepted" condition when it is
+	// set to false because all CACertificateRefs of the
+	// BackendTLSPolicy are invalid.
+	BackendTLSPolicyReasonNoCACertificates v1alpha2.PolicyConditionReason = "NoCACertificates"
+)
+
+const (
+	// This condition indicates whether the controller was able to resolve all
+	// object references for the BackendTLSPolicy.
+	//
+	// Possible reasons for this condition to be True are:
+	//
+	// * "ResolvedRefs"
+	//
+	// Possible reasons for this condition to be False are:
+	//
+	// * "InvalidCACertificateRef"
+	// * "InvalidKind"
+	//
```

Comment:
Isn't this condition also false for reason `BackendTLSPolicyReasonNoCACertificates` ?

Reply:
`NoCACertificates` or `NoValidCACertificate` is used with `Accepted: false` and not `ResolvedRefs: false`